sdk/webpubsub/Azure.Messaging.WebPubSub/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0002:DO ensure all service methods, both asynchronous and synchronous, take an optional CancellationToken parameter called cancellationToken.", Justification = "CancellationToken can be passed through RequestOptions")]
```

Comment:
Opened https://github.com/Azure/azure-sdk-tools/issues/1715 so we don't lose track of fixing the analyzer here.

Reply:
Though, to be clear. I think this is triggered by my experiments in not having CT parameters in some helper overloads.